spinoso-time/src/time/tzrs/offset.rs

```diff
@@ -172,10 +179,18 @@ impl From<&str> for Offset {
             "W" => Self::fixed(-10 * SECONDS_IN_HOUR),
             "X" => Self::fixed(-11 * SECONDS_IN_HOUR),
             "Y" => Self::fixed(-12 * SECONDS_IN_HOUR),
+            // ```console
```

Comment:
Ah Ruby REPL here is what you meant - sorry, I thought you meant in artichoke-backend, but this is clear and makes way more sense!